src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -274,7 +274,9 @@ class ProductConfigFormController {
     this.updateQueryParam({ key: giveGiftParams.amount, value: amount })
   }
 
-  changeCustomAmount (amount, retainCoverFees) {
+  changeCustomAmount (stringAmount, retainCoverFees) {
```

Comment:
I forget if this is called before or after the custom validators.

Reply:
Looks like this change does alter the way the custom amount field works in that it no longer allows `$` or `,` to be added. This is not necessarily a bad change, but it is a change of functionality on the form itself vs just fixing the analytics.

Reply:
Good catch. I'm trying a different approach that should preserve the the punctuation so that `itemConfig.AMOUNT` can consistently be a number throughout the application. But if that doesn't work, I'll just do the parsing in the datalayer.

Reply:
Note, too, that `itemConfig.AMOUNT` is also coming from the API, probably as a string, but I'm not 100% sure.